livekit-api/livekit/api/__init__.py

```diff
@@ -22,6 +22,6 @@
 from livekit.protocol import models
 from livekit.protocol import room
```

Comment:
we should re-export the other types too. and maybe change them to this namespace? i.e. so users can call `api.StartWebEgressRequest()` ? versus having to know if a symbol is from room or egress or models